cfn-templates/msindexer.yaml

```diff
@@ -65,33 +65,44 @@ Resources:
                   - 'dynamodb:*'
                 Resource: !GetAtt YTMediaDDBQueueTable.Arn
   
-  PytubeLambdalayer:
-    Type: AWS::Lambda::LayerVersion
+  YTDLPLambdalayer:
+    Type: "AWS::Lambda::LayerVersion"
     Properties:
-      Content:
-        S3Bucket: '<ARTIFACT_BUCKET_TOKEN>'
-        S3Key: !Join
-          - ''
-          - - '<ARTIFACT_PREFIX_TOKEN>'
-            - '<PYTUBELLAYER_ZIPFILE>'
-      Description: Layer with pytube package required to download YT Videos
-      LayerName: pytube-llayer
+      Content: ../layers/yt_dlp
+      CompatibleRuntimes:
+        - python3.11      
+      Description: Layer with yt_dlp package required to download YT Videos
       LicenseInfo: MIT
+
+#  QBusinessModelLayer:
```

Comment:
We don't need QBisinessLayer for this release, right?

Reply:
No, I have removed the commented line to avoid confusion